src/storage-preview/azext_storage_preview/_transformers.py

```diff
@@ -0,0 +1,133 @@
+            new_entry['ExposedHeaders'] = ', '.join((x for x in rule.exposed_headers))
+            new_entry['AllowedHeaders'] = ', '.join((x for x in rule.allowed_headers))
+            new_entry['MaxAgeInSeconds'] = rule.max_age_in_seconds
+            new_result.append(new_entry)
+    return new_result
+
+
+def transform_entity_query_output(result):
+    from collections import OrderedDict
+    new_results = []
+    ignored_keys = ['etag', 'Timestamp', 'RowKey', 'PartitionKey']
+    for row in result['items']:
+        new_entry = OrderedDict()
+        new_entry['PartitionKey'] = row['PartitionKey']
+        new_entry['RowKey'] = row['RowKey']
+        other_keys = sorted([x for x in row.keys() if x not in ignored_keys])
+        for key in other_keys:
+            new_entry[key] = row[key]
+        new_results.append(new_entry)
+    return new_results
+
+
+def transform_entities_result(result):
+    for entity in result.items:
+        transform_entity_result(entity)
+    return result
+
+
+def transform_entity_result(entity):
+    for key in entity.keys():
+        entity_property = entity[key]
+        if hasattr(entity_property, 'value') and isinstance(entity_property.value, bytes):
+            entity_property.value = base64.b64encode(entity_property.value).decode()
+    return entity
+
+
+def transform_logging_list_output(result):
+    from collections import OrderedDict
+    new_result = []
+    for key in sorted(result.keys()):
+        new_entry = OrderedDict()
+        new_entry['Service'] = key
+        new_entry['Read'] = str(result[key]['read'])
+        new_entry['Write'] = str(result[key]['write'])
+        new_entry['Delete'] = str(result[key]['delete'])
+        new_entry['RetentionPolicy'] = str(result[key]['retentionPolicy']['days'])
+        new_result.append(new_entry)
+    return new_result
+
+
+def transform_metrics_list_output(result):
+    from collections import OrderedDict
+    new_result = []
+    for service in sorted(result.keys()):
+        service_name = service
+        for interval in sorted(result[service].keys()):
+            item = result[service][interval]
+            new_entry = OrderedDict()
+            new_entry['Service'] = service_name
+            service_name = ''
+            new_entry['Interval'] = str(interval)
+            new_entry['Enabled'] = str(item['enabled'])
+            new_entry['IncludeApis'] = str(item['includeApis'])
+            new_entry['RetentionPolicy'] = str(item['retentionPolicy']['days'])
+            new_result.append(new_entry)
+    return new_result
+
+
+def create_boolean_result_output_transformer(property_name):
+    def _transformer(result):
+        return {property_name: result}
+
+    return _transformer
+
+
+def transform_storage_list_output(result):
+    if getattr(result, 'next_marker', None):
+        logger.warning('Next Marker:')
+        logger.warning(result.next_marker)
+    return list(result)
+
+
+def transform_url(result):
```

Comment:
seems this is not used anywhere, and if it was why would url contain extra characters?